common/lib/driver-definitions/src/storage.ts

```diff
@@ -98,6 +98,9 @@ export interface IDocumentStorageServicePolicies {
     // If this policy is provided, it tells runtime on ideal size for blobs
     // Blobs that are smaller than that size should be aggregated into bigger blobs
     readonly minBlobSize?: number;
+
+    // This policy is required to be defined to run GC.
```

Comment:
Can you add details of what this policy means, i.e., it tells the runtime that the driver will not use cached snapshots that are older than this, or something along those lines.

Also, this does not have to be defined to run GC. If it is undefined, we can assume some default. So, we should not mention it.